LSP-rust-analyzer.sublime-settings

```diff
@@ -1,6 +1,7 @@
 {
+	"version": "2021-06-07",
```

Comment:
BTW. I'm not sure it's that good of an idea to expose this option.
Nice as user will be able to upgrade faster or downgrade if there is some incompatibility but at the same time it will be very easy for the user to forgot and leave his override which will prevent him from getting updated versions when package is upgraded.

Reply:
Agreed. I removed the ability to choose which server version you can download. The user can always download a binary themselves manually and override the `"command"`.